styles.css

```diff
@@ -82,3 +82,181 @@ nav {
   border: 1px solid #dfe1e6;
   margin: 0 auto;
 }
+
+.projects {
+  padding: 24px;
+}
+
+.work-sec-title {
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  padding-bottom: 24px;
+  text-align: center;
+}
+
+.indicator {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 62px;
+}
+
+.projects-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 24px;
+}
+
+.project-card {
+  height: 494px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+}
+
+.project-card--thumb {
+  height: 260px;
+  background-color: #ebf0ee;
+}
+
+.project-card--content {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 24px;
+}
+
+.project-title {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+}
+
+.project-tools {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+.project-tools li {
```

Comment:
- [ ] You can decrease the padding/margin on the tools section so your design matches the Figma design 

![Screenshot_20220526_185509](https://user-images.githubusercontent.com/25227022/170547095-1e508583-e333-44bf-800b-611213378fcc.png)

Reply:
Thank you for reviewing my project, I will make the necessary corrections as suggested.